inc/Engine/Media/AboveTheFold/WarmUp/APIClient.php

```diff
@@ -24,7 +26,7 @@ public function add_to_atf_queue( string $url ): array {
 
 		$config = [
 			'optimization_list' => '',
-			'is_home'           => Utils::is_home( $url ),
+			'is_home'           => $is_home,
```

Comment:
Is this change necessary?

Reply:
I'm failing to understand the difference it makes, it's the same code assigned to a variable?

Reply:
ah yes right, I see now, all good